Clarify key rotations test cases by using keywords

MVrachev opened the pull request:
**Description of the changes being introduced by the pull request**:

As I mentioned in my comment here https://github.com/theupdateframework/python-tuf/issues/1607#issuecomment-953160724 while I was reviewing #1635 it was a little hard for me to remember the placement of the arguments for RootVersion considering they are 15 root rotations tested and the dataset becomes a larger piece of code.

That's why I suggest using keywords whenever possible to do that without splitting the `RootVersion` instance creation into two lines.

Signed-off-by: Martin Vrachev <mvrachev@vmware.com>

**Please verify and check that the pull request fulfills the following
requirements**:

- [ ] The code follows the [Code Style Guidelines](https://github.com/secure-systems-lab/code-style-guidelines#code-style-guidelines)
- [ ] Tests have been added for the bug fix or new feature
- [ ] Docs have been added for the bug fix or new feature

Commit:
Clarify key rotations test cases by using keywords

Signed-off-by: Martin Vrachev <mvrachev@vmware.com>

jku commented on tests/test_updater_key_rotations.py:
```diff
@@ -87,71 +87,71 @@ def _run_refresh(self) -> None:
         "1-of-1 key rotation fail: not signed with old key": [
-            RootVersion([1], 1, [1]),
-            RootVersion([2], 1, [2, 3, 4], UnsignedMetadataError),
+            RootVersion(keys=[1], threshold=1, sigs=[1]),
+            RootVersion([2], 1, sigs=[2, 3, 4], res=UnsignedMetadataError),
```
this variance is IMO making it harder to read than it was -- now it's really hard to see what changed between these two versions whereas in the original that's trivial. 

I think adding the names makes sense, but only if it doesn't impact the ability to notice changes.

jku replied:
maybe changing res to a boolean would help (unlikely that we'll need other errors)... or maybe we can allow longer lines for this part of the code somehow

MVrachev replied:
I think it's best if we add keywords for all cases and disable long lines warning for those where we have problems.
That way the code will be consistent. 
The long cases are little hard to follow now, but with added keywords they will be much better.

jku replied:
a potential issue with that plan is that we currently enforce line length with 3 (!) different tools: two of those would probably require a rule disable here. Tools are:
* black: this makes sense, it actually formats the code
* isort: also makes sense: it modifies imports so needs to know line length (isort is also not a problem in this case since it only cares about imports)
* pylint: this may be unnecessary? why does pylint also need to check the length?

jku replied:
> * pylint: this may be unnecessary? why does pylint also need to check the length?

It is unnecessary, we can disable the whole "format" checker. Even pylints own pylintrc does that  https://github.com/PyCQA/pylint/blob/main/pylintrc#L80 :joy:

MVrachev replied:
pylint format option is disabled in #1659.

Will update the pr after it's merged.

MVrachev commented:
#1659  was merged and I updated the pr to use keywords in all test cases.
@jku do you think I should rename `res` back to `results` and `sigs` back to `signatures`?

jku reviewed (approved):
LGTM. Is black going to accept these without complaints when you enable black?

Commit:
disable black for part of key rotations tests

In order to make the `RootVersion` test cases understandable it's
better if we use keywords and don't split the different RootVersions
into multiple lines.

Signed-off-by: Martin Vrachev <mvrachev@vmware.com>

MVrachev commented:
Ohh... I forgot to run black on that change.
It's complaining...
I disabled  `black` with `# fmt: off` for the test cases and then enabled it again with `# fmt: on` after them.
